moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -227,22 +185,56 @@ const ProjectsListViewTable = () => {
         />
         {/*Main Table Body*/}
         <Paper className={classes.paper}>
-          <Box mt={3}>
-            {!data && !error ? (
-              <CircularProgress />
-            ) : data && data.project_list_view ? (
-              <Card className={classes.root}>
-                <MaterialTable
-                  onChangeColumnHidden={handleColumnChange}
-                  data={data.project_list_view}
-                  columns={columns}
-                  title=""
-                  options={tableOptions}
-                  components={tableComponents}
-                />
-              </Card>
-            ) : (
-              <span>{error ? error : "Could not fetch data"}</span>
+          <Box sx={{ height: "600px", padding: "14px", marginTop: "24px" }}>
```

Comment:
we have the sx prop in a few of our components sprinkled about, I'm torn between leaving this here for legibility vs moving it into a classes.styleName